chain/consensus/mir/wal.go

```diff
@@ -0,0 +1,17 @@
+package mir
```

Comment:
We don't need this file yet, can be deleted for now.

Reply:
Fixed